docs/how-to-guides/quick-start.md

```diff
@@ -1,11 +1,11 @@
 # Quick start
 
-This guide will walk you through the fist step using of prompt flow code-first experience.
+This guide will walk you through the first steps of the prompt flow code-first experience.
 
 **Prerequisite** - To make the most of this tutorial, you'll need:
-- Know how to program with Python :)
+- Python programming knowledge
```

Comment:
could we merge this change into PR 3461? so we have 1 instead of 2 PR for the same purpose

Reply:
I’m okay with that but I’m on vacation so it will be a while before I’m at a keyboard and can do that.

Reply:
Oh looks like the other one was merged already

Reply:
Yes. The original one is merged already. I think we can just merge this one also.